### ISSUE TITLE
Do you use RL-based algorithm to find the best hyperparameters?

### ISSUE THREAD

**Issue by Z7zuqer** [SCUT-AILab/DCP#3]
learning rate, pruning ratio, e.g.

**Comment by liujingcs** [comment]
No, we do not try that.

**Comment by Z7zuqer** [comment]
Thx.
